README.md

```diff
@@ -1,7 +1,7 @@
 9wm
 ====
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½.
+9wm is an X11 window manager inspired by the Plan 9 window manager rio.
```

Comment:
Ah ok 
Would you rather I change it back or just change it to "8 1/2 / rio" or something like that?